apps/meteor/client/views/admin/users/hooks/useFilterActiveUsers.ts

```diff
@@ -1,9 +1,6 @@
 import type { IUser } from '@rocket.chat/core-typings';
 
-export const useFilterActiveUsers = (
-	users: Pick<IUser, '_id' | 'username' | 'name' | 'status' | 'roles' | 'emails' | 'active' | 'avatarETag'>[] | undefined,
```

Comment:
This typing change is not correct, rest-typings for users.list have been set up to only return this subset of params of the users object, changing it to `Partial<IUser>` will make all fields in the User Object optional and cause errors like the one in the line 180 of `UsersTable.ts` in which the `_id` of the users returned to `filteredUsers` can be undefined whilst the `_id` of the users that `UsersTableRow` expect can not. This happens because IUser sets `_id` as mandatory and this should be respected.